packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -542,7 +547,7 @@ class FileCollection extends DocumentCollection {
      * (no size limit since we can use the body for that) and after we use the ID.
      */
     let metadataId
-    let path = uri`/files/${fileId}?Name=${sanitizedName}&Type=file&Executable=${executable}`
+    let path = uri`/files/${fileId}?Name=${sanitizedName}&Type=file&Executable=${executable}&Encrypted=${encrypted}`
```

Comment:
```suggestion
    let path = uri`/files/${fileId}?Name=${sanitizedName}&Type=file&Executable=${executable}&encrypted=${encrypted}`
```
????

Reply:
oups, it looks like a behaviour in cozy-client. 🤔 Strange. But i resolve

Reply:
!!!! ?
The stack expects the query-string params to be like this: https://github.com/cozy/cozy-stack/blob/60193a284672216f502804862f44665771f35ffa/docs/files.md#query-string-1